be/src/exec/workgroup/work_group.h

```diff
@@ -244,6 +252,14 @@ class WorkGroupManager {
 
     std::unique_ptr<WorkerOwnerManager> _driver_worker_owner_manager;
     std::unique_ptr<WorkerOwnerManager> _scan_worker_owner_manager;
+
+    bool _init_metrics = false;
+    std::unordered_map<std::string, int128_t> _wg_metrics;
+
+    std::unordered_map<std::string, starrocks::DoubleGauge*> _wg_cpu_limit_metrics;
```

Comment:
Why use the wg_name instead of id to index metrics?

Reply:
The same group name may have different ids, for example, delete and then create, but it is the same resource group for users, and metrics are also identified by name.